### PR TITLE
Stripped Dispersy from Gumby

### PR DESCRIPTION
**Pull request by devos50**

### CODE REVIEW

File: docs/hello_world.rst [hunk 1]
```diff
@@ -1,246 +0,0 @@
-The purpose of this document is to showcase creation of a basic Gumby experiment.
```
Comment (qstokkink) [inline docs/hello_world.rst]: Why was my beautiful documentation removed? 😢

Comment (devos50) [reply docs/hello_world.rst]: Because it is completely based on Dispersy. I am planning on rewriting it and I should probably do that tomorrow.

Comment (qstokkink) [reply docs/hello_world.rst]: 98% of this file is about general Gumby configuration, throwing it all out seems like a waste.

Comment (devos50) [reply docs/hello_world.rst]: You're right, I didn't read through it properly. I made a few changes to make the last section use IPv8 instead.

**Comment by devos50** [comment]
For some reason, libtorrent is not working correctly in a newly setup DAS5 environment. I will look into it and fix this in another PR. The current DAS5 environments are working correctly and as expected.

**Review by qstokkink — APPROVED** [review]

**Pull request merged**